packages/suite/src/hooks/coinjoin/useBlockedCoinjoinResume.ts

```diff
@@ -0,0 +1,39 @@
+import { useSelector } from '@suite-hooks';
+import { selectDeviceState } from '@suite-reducers/suiteReducer';
+import { selectIsCoinjoinBlockedByTor } from '@wallet-reducers/coinjoinReducer';
+import { getIsCoinjoinOutOfSync } from '@wallet-utils/coinjoinUtils';
+
+export const useBlockedCoinjoinResume = () => {
+    const isCoinJoinBlockedByTor = useSelector(selectIsCoinjoinBlockedByTor);
+    const { selectedAccount, online } = useSelector(state => ({
+        selectedAccount: state.wallet.selectedAccount,
+        online: state.suite.online,
+    }));
+    const isAccountOutOfSync = getIsCoinjoinOutOfSync(selectedAccount);
+    const deviceStatus = useSelector(selectDeviceState);
+    const isDeviceDisconnected = deviceStatus !== 'connected';
```

Comment:
In mine I was checking the `available` flag, in this case they are the same I suppose?